### DISCUSSION TITLE
v2版本下搜索栏样式并不好看

### DISCUSSION THREAD

**Discussion by FCSDFSDVXDFG** [the1812/Bilibili-Evolved#2195] [综合讨论 / General]
v1下的搜索栏
![Snipaste_2021-08-23_19-00-46](https://user-images.githubusercontent.com/65455414/130437157-a4f54024-54e0-4f58-ad9c-f3ad1da3cac3.png)
v2下的搜索栏
![Snipaste_2021-08-23_19-02-47](https://user-images.githubusercontent.com/65455414/130437178-931a75e4-169a-437a-8eed-742700b8dcd9.png)
建议把v2下的搜索栏改回v1的样式

**Comment by the1812** [comment]
这样?
![image](https://user-images.githubusercontent.com/26504152/130441918-b3d99543-1f21-4cf9-afc0-c8efd6eb5656.png)

**Comment by Geeyun-JY3** [comment]
话说为啥要从放大镜改成箭头（

**Reply by the1812** [reply]
因为不一定是搜索了, 也能执行一些命令
